veloroutes_voies_vertes/processing/covadis/export_covadis.py

```diff
@@ -182,10 +182,15 @@ def to_shapefile(self, context, table, dpt, dirname, uri, geom, charger):
         options.driverName = "ESRI ShapeFile"
         options.fileEncoding = "utf-8"
 
+        # construction du répertoire
+        if not os.path.exists(dirname):
+            os.mkdir(dirname)
+        dirname = os.path.join(dirname, filename + ".shp")
```

Comment:
```suggestion
        file_path = os.path.join(dirname, filename + ".shp")
```
Il s'agit désormais du chemin du fichier.